dnase/parseSamtoolsGenotypesToBedFiles.pl

```diff
@@ -166,6 +175,20 @@ sub parseGenotype {
         if (($GT1 ne $GT2) or ($GQ1 != $GQ2)) {
             die "Wrong parsing assumptions for VCF genotype ($genotype)\n";
         }
+    } elsif ($format eq "GT:GL:GQ:FT:RCL:RC:RCR:RDCN:DR:DV:RR:RV") {
+        my ($GT1,$GL1,$GQ1,$FT1,$RCL1,$RC1,$RCR1,$RDCN1,$DR1,$DV1,$RR1,$RV1) = split /\:/, $genotype;
+        ## bcftools => FORMAT=<ID=DP,Number=1,Type=Integer,Description="Number of high-quality bases">
+        #$readDepth = $DP;
+        ## delly => FORMAT=<ID=RC,Number=1,Type=Integer,Description="Raw high-quality read counts or base counts for the SV"
+        # FIXME: Make sure this is correct, there are other paramaters related to depth
+        $readDepth = $RC1;
+        $qualityScore = $GQ1;
+        $GT = $GT1;
+        ## bcftools => FORMAT=<ID=PL,Number=G,Type=Integer,Description="List of Phred-scaled genotype likelihoods">
+        # $PL = $PL1;
+        ## delly => FORMAT=<ID=GL,Number=G,Type=Float,Description="Log10-scaled genotype likelihoods for RR,RA,AA genotypes">
+        # FIXME: Make sure this is correct, I dont like using PL to store GL
+        $PL = $GL1
```

Comment:
I think this should probably be PL="NA"; I think PL is only used for output

Reply:
I will update this in the next commit